apps/ledger-live-desktop/src/renderer/screens/platform/App.jsx

```diff
@@ -20,17 +20,29 @@ type Props = {
     url: string,
   },
   appId?: string,
+  location: {
+    hash: string,
+    params: {
+      [key: string]: string,
+    },
+    pathname: string,
+    search: string,
+  },
 };
 
-export default function PlatformApp({ match, appId: propsAppId }: Props) {
+export default function PlatformApp({ match, appId: propsAppId, location }: Props) {
   const history = useHistory();
-  const { state: urlParams, search } = useLocation();
+  const { params: urlParams, search, pathname } = location;
   const appId = propsAppId || match.params?.appId;
-
+  const sufix = pathname.substr(
+    pathname.indexOf(match.params.appId) + match.params.appId.length + 1,
+  );
   const localManifest = useLocalLiveAppManifest(appId);
   const remoteManifest = useRemoteLiveAppManifest(appId);
+  const manifest = JSON.parse(JSON.stringify(localManifest || remoteManifest));
 
-  const manifest = localManifest || remoteManifest;
+  const dappUrl = manifest.params.dappUrl.split("?");
```

Comment:
I think the snapshot tests are failing on this line https://github.com/LedgerHQ/ledger-live/pull/2179#issuecomment-1363139928